docs/helpers.md

```diff
@@ -0,0 +1,37 @@
+# Helpers
+
+Regardless of the way NixVim is used (as a home-manager module, a nixos module, or as a standalone module),
+helpers can be included in the same way.
+
+You can simply use:
+
+```nix
+{
+    helpers,
+    ...
+}: {
+    # Your config
+}
+```
+
+A certain number of helpers are defined that can be useful:
+
+- `helpers.emptyTable`: An empty lua table `{}` that will be included in the final lua configuration.
+
+- `helpers.mkRaw str`: Write the string `str` as raw lua in the final lua configuration.
```

Comment:
Maybe we could give the equivalence here: `.__raw = "value"` and `__empty = <whatever>`.

Reply:
Yes!